audit-triplestore/src/main/java/org/fcrepo/camel/audit/triplestore/AuditSparqlProcessor.java

```diff
@@ -114,22 +116,24 @@ public void process(final Exchange exchange) throws Exception {
     private static final String NODE_REMOVED = REPOSITORY + "NODE_REMOVED";
     private static final String PROPERTY_CHANGED = REPOSITORY + "PROPERTY_CHANGED";
 
+    private static final String EMPTY_STRING = "";
+
     /**
      * Convert a Camel message to audit event description.
-     * @param m JMS message produced by an audit event
+     * @param message JMS message produced by an audit event
```

Comment:
I thought I had fixed this already, but this should say it's a Camel message, not JMS.